crates/graphql/src/schema.rs

```diff
@@ -897,13 +897,20 @@ impl QueryRoot {
     fn nfts(
         &self,
         context: &AppContext,
-        #[graphql(description = "Filter on creator address")] creators: Vec<String>,
+        #[graphql(description = "Filter on owner address")] owners: Option<Vec<String>>,
+        #[graphql(description = "Filter on creator address")] creators: Option<Vec<String>>,
         #[graphql(description = "Filter on attributes")] attributes: Option<Vec<AttributeFilter>>,
-    ) -> Vec<Nft> {
+    ) -> FieldResult<Vec<Nft>> {
+        if owners.is_none() && creators.is_none() {
+            return Err(FieldError::new(
+                "No filter provided! Please provide atleast one of the filter",
```

Comment:
```suggestion
                "No filter provided! Please provide at least one of the filters",
```

Reply:
@imabdulbasit i see you resolved this, did the fix get committed?

Reply:
I pushed a commit

Reply:
Changeset doesn't show it was fixed, are you sure?

Reply:
https://github.com/holaplex/solana-indexer/pull/128/commits/be6664be3d22cbf6f4b61731ef4b7d3780118467

Reply:
:+1: aight idk what happened on github's end lol